contrib/localdev/bootstrap-kind-cluster.sh

```diff
@@ -1,13 +1,13 @@
 #! /bin/bash
 
 error() {
-    echo $@ 1>&2
+    echo "$@" 1>&2
     exit 1
 }
 
 set -eu
 
-PROJECT_ROOT="$(realpath $(dirname "${BASH_SOURCE[0]}")/../..)"
+PROJECT_ROOT="$(realpath "$(dirname "${BASH_SOURCE[0]}")"/../..)"
```

Comment:
Unrelated to this PR but `git rev-parse --show-toplevel` would be more resilient.

Reply:
I'll update it too